addons/purchase_requisition_sale_stock/tests/test_purchase_requisition_sale_stock.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import TransactionCase
```

Comment:
Don't create a module for a test.
Instead put it in purchase_requistion_stock.
Usecase:
Warehouse 2 steps
Trigger replenishment
Create an alternative PO
Choose the alternative PO and confirm
Validate the receipt
Check the internal transfer could be reserve (the link between stock.move is correctly done)